apps/admin-gui/src/app/shared/components/add-owner-dialog/add-owner-dialog.component.ts

```diff
@@ -0,0 +1,53 @@
+import { Component, OnInit } from '@angular/core';
+import { MatDialogRef } from '@angular/material/dialog';
+import { NotificatorService } from '@perun-web-apps/perun/services';
+import { OwnersManagerService } from '@perun-web-apps/perun/openapi';
+import { TranslateService } from '@ngx-translate/core';
+import { FormControl, Validators } from '@angular/forms';
+
+@Component({
+  selector: 'perun-web-apps-add-owner-dialog',
+  templateUrl: './add-owner-dialog.component.html',
+  styleUrls: ['./add-owner-dialog.component.scss']
+})
+export class AddOwnerDialogComponent implements OnInit {
+
+  constructor(
+    private dialogRef: MatDialogRef<AddOwnerDialogComponent>,
+    private notificator: NotificatorService,
+    private ownersManagerService:OwnersManagerService,
+    private translate: TranslateService
+  ) {
+    translate.get('DIALOGS.ADD_OWNER.SUCCESS').subscribe(value => this.successMessage = value);
+  }
+
+  successMessage: string;
+  loading: boolean;
+
+  nameCtrl: FormControl;
+  contactCtrl: FormControl;
+  type = '1';
+
+  ngOnInit() {
+    this.nameCtrl = new FormControl(null, [Validators.required, Validators.pattern('^[\\w.-]+( [\\w.-]+)*$')]);
+    this.contactCtrl = new FormControl(null, [Validators.required, Validators.pattern('^[\\w.-]+( [\\w.-]+)*$')]);
```

Comment:
This is probably wrong regexp for this form control, because if I write the symbol "@" into the contact form fielad, I will get this form field error.
![create_owner](https://user-images.githubusercontent.com/70371638/110799905-f355b200-827b-11eb-847c-d2a7422dd200.png)